src/gui/editors/Editor.cpp

```diff
@@ -155,6 +154,16 @@ void Editor::closeEvent(QCloseEvent * event)
 	event->ignore();
  }
 
+ void Editor::keyPressEvent(QKeyEvent * ke)
```

Comment:
> For pointers or references, always use a single space between the type and '*' or '&', but no space between the '*' or '&' and the variable name

Source: [Qt Coding Style](https://wiki.qt.io/Qt_Coding_Style#Spaces_within_code) since it is not explicitly overidden in the LMMS coding conventions

```suggestion
 void Editor::keyPressEvent(QKeyEvent *ke)
```